### PR TITLE
LRAC-13921 | master

### PR DESCRIPTION
**Pull request by leilanyaragao**
Jira ticket: https://issues.liferay.com/browse/LRAC-13921

### CODE REVIEW

**Comment by leilanyaragao** [comment]
ci:test:sf

**Comment by leilanyaragao** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/toolbar/index.jsx [hunk 1]
```diff
@@ -485,7 +485,7 @@ export default class Toolbar extends React.Component {
 							<SubnavTbar.Item>
 								<ClayButton
 									className='button-root'
-									display='secondary'
+									displayType='tertyary'
```
Comment (ilzamcmed) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/toolbar/index.jsx:488]:
```suggestion
displayType='link'
```

**Review by ilzamcmed — CHANGES_REQUESTED** [review]

**Comment by leilanyaragao** [comment]
ci:test:sf

**Comment by leilanyaragao** [comment]
ci:test:relevant

**Review by ilzamcmed — APPROVED** [review]
LGTM! :rocket:

**Pull request closed without merging**